Editor/Window/ConnectToFleetInput.cs

```diff
@@ -170,6 +170,11 @@ private async Task UpdateFleetMenu()
                             textProvider.Get(Strings.AnywherePageConnectFleetStatusActive));
                     }
                 }
+                else
```

Comment:
Do you not need to set the _fleetState when the fleet received is not `null`? Even if that's the case I'd add it for consistency.

Reply:
I have moved this check to the only place that it is used.